sqlgen/db.go

```diff
@@ -55,17 +53,26 @@ func NewDB(conn *sql.DB, schema *Schema) *DB {
 				return nil, err
 			}
 			clause, args = selectQuery.ToSQL()
+			span.SetTag("query", clause)
+			span.SetTag("batchItemCount", len(items))
+			if marshaledItems, err := json.Marshal(filters); err == nil {
+				span.SetTag("batchItems", string(marshaledItems))
```

Comment:
Ah sorry, I meant to update that comment before posting. I think it should be fine to log the filters, but I don't think it's necessary. We won't gain visibility into how long each filter takes on the SQL side.